examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -241,15 +207,18 @@ def evaluate(model, X, y, cv):
     )
 
 
-evaluate(gbrt_pipeline, X, y, cv=ts_cv)
+evaluate(gbrt, X, y, cv=ts_cv)
 
 # %%
 # This model has an average error around 4 to 5% of the maximum demand. This is
-# quite good for a first trial without any hyper-parameter tuning! We just had
-# to make the categorical variables explicit. Note that the time related
-# features are passed as is, i.e. without processing them. But this is not much
-# of a problem for tree-based models as they can learn a non-monotonic
-# relationship between ordinal input features and the target.
+# quite good for a first trial without any feature engineering nor
+# hyper-parameter tuning! We just had to make the categorical variables
+# explicit by using the "category" dtype in the pandas dataframe.
```

Comment:
@betatim what do you think about the following?

```suggestion
# hyper-parameter tuning! We just had to ensure that we were using the
# "category" dtype in the pandas dataframe for all the categorical
# variables.
```

Reply:
Sounds good. The point that I think is important to get across to the reader is that "you don't have to do anything! It just works, as if by magic! (if your data frame columns are already marked as categorical)".